developers_chamber/ecs_utils.py

```diff
@@ -45,48 +46,36 @@ def get_log_events(log_group, log_stream, region):
     return resp['events']
 
 
-def register_new_task_definition(task_definition, image, region, ecs_client=None):
+def register_new_task_definition(task_definition_name, images, region, ecs_client=None):
+    images = json.loads(images)
```

Comment:
pozor tady je také chyba, neměl bys měnit význam vstupní proměnné. Mělo by set o jmenovat třeba images_data

Reply:
co když tam pošlel špatný json tak to padne na hodně blbé chybě. Měly bychom vrátit tu click exception